app/src/main/java/com/azuresamples/msalandroidapp/MultipleAccountModeFragment.java

```diff
@@ -298,6 +303,10 @@ public void onCancel() {
 
     /**
      * Make an HTTP request to obtain MSGraph data
+     *
+     * The sample is using the global service cloud as a default.
+     * If you're developing an app for sovereign cloud users, please change the Microsoft Graph Resource URL accordingly.
```

Comment:
It would be good to call out here that this statement is referring to changing the value set on the `graphResourceTextView`